lib/MusicBrainz/Server/Controller/User.pm

```diff
@@ -465,7 +465,7 @@ sub tag : Chained('load') PathPart('tag') Args(1)
     );
 }
 
-sub privileged : Path('/privileged')
```

Comment:
I don't think we need/ought to `RequireAuth` for `/privileged`.

Reply:
That was completely unnecessary and is reverted in https://github.com/metabrainz/musicbrainz-server/pull/536/commits/69d3c67b20df402de75ae1e0a9ae73436555e2e1